uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeaveRequest.php

```diff
@@ -267,7 +267,7 @@ private static function validateTOILPastDays($params) {
    */
   private static function validateTOILToAccruedAmountIsValid($params) {
     $absenceType = AbsenceType::findById($params['type_id']);
-    $unlimitedAccrual = empty($absenceType->max_leave_accrual) && $absenceType->max_leave_accrual != 0;
+    $unlimitedAccrual = empty($absenceType->max_leave_accrual) && $absenceType->max_leave_accrual !== 0;
```

Comment:
I tried requesting a TOIL when the max_leave_accrual was NULL and  could not. Discovered that the variable `$unlimitedAccrual` has a value of false. That was because NULL was type casted to Zero, So I had o change the comparison operator.